app/models/case/sar/offender.rb

```diff
@@ -47,7 +47,6 @@ def searchable_fields_and_ranks
                  subject_type: :string,
                  third_party_relationship: :string,
                  third_party: :boolean,
-                 requester_reference: :string,
```

Comment:
Why has this been removed?

Reply:
@willmcb  sorry for confusion!, requester_reference has been duplicated (at line 44)  from master branch,  I always saw this warning message from my console, so.... I removed